analyses/tmb-compare/compare-tmb-calculations.Rmd

```diff
@@ -122,29 +123,59 @@ all_tmb <- tmb_ccdl_maftools %>%
                 mutation_count_d3b = count)
 ```
 
-## Percent Differences in TMB
+## Fraction differences in TMB
```

Comment:
I didn't even say anything! (The system works!)

Reply:
I knew it was probably eating you up.